service/src/main/java/uk/gov/hmcts/reform/fpl/model/Children.java

```diff
@@ -36,6 +36,7 @@ public Children(Child firstChild, Child... additionalChildren) {
     }
 
     @JsonIgnore
+    @Size(min = 1, message = "Tell us the names of all children in the case")
     public List<Child> getAllChildren() {
```

Comment:
Don't you need `List<@Valid Child>`? I cannot remember whether it happens in an implicit way.